Library/ViewModels/PledgeViewModel.swift

```diff
@@ -497,6 +497,7 @@ public class PledgeViewModel: PledgeViewModelType, PledgeViewModelInputs, Pledge
     )
     .map { paymentData, token in
       (
+        nil,
```

Comment:
Do we validate checkout in the normal pledge flow? Because if we do, we probably need to do so for apple pay here, too.

Reply:
Nope. ValidateCheckout is just for late pledge as of now.